problems/countWords.js

```diff
@@ -14,7 +14,17 @@
 * returns: {}
 */
 
-function countWords() {
+const countWords = (arr) => {
+    let result = {};
+    for(let i = 0; i < arr.length; i++){
+        let words = arr[i];
+        if(result[words] !== undefined){
+            result[words] += 1;
+        }else{
+            result[words] = 1;
+        }
+    }
```

Comment:
Good work,  whenever we need loop through an entire array and do some side effect logic like what you did here for the `result={}` object,  is a great indication to use the `forEach` method for arrays.